src/api/namespaces/neftymarket/handlers/assets.ts

```diff
@@ -3,10 +3,23 @@ import { NeftyMarketContext } from '../index';
 import { fillAssets } from '../../atomicassets/filler';
 import { buildAssetFillerHook, formatListingAsset } from '../format';
 import { getRawAssetsAction } from '../../atomicassets/handlers/assets';
+import QueryBuilder from '../../../builder';
+import {filterQueryArgs} from '../../validation';
 
 export async function getMarketAssetsAction(params: RequestValues, ctx: NeftyMarketContext): Promise<any> {
+    const args = filterQueryArgs(params, {
+        symbol: {type: 'string', default: null},
```

Comment:
why not default it to a configuration constant or something? doesn't work?

Reply:
Because it depends on the chain

Reply:
yeah I meant like an environment variable

Reply:
It can be a configuration.